distribution/docs/src/main/resources/_contents/_app-whitelists/admin-whitelist-contents.adoc

```diff
@@ -3,7 +3,6 @@
 
 The following packages have been exported by the ${ddf-admin} application and are approved for use by third parties:
 
-* `org.codice.ddf.ui.admin.api`
 * `org.codice.ddf.ui.admin.api.module`
```

Comment:
We could. Basically that package has no business being whitelisted, because it was only for mbean interfaces anyways, which are all internal. The api.module one under it should not have been whitelisted either and I'm not really sure how it ended up on this list. All of the admin console backend has always been internal to us.